src/schema/v2/previewSavedSearch/searchCriteriaLabel.ts

```diff
@@ -518,25 +512,13 @@ async function getPartnerLabels(partnerIDs: string[], partnerLoader) {
 
 async function getArtistSeriesLabels(
   artistSeriesIDs: string[],
-  gravityGraphQLLoader
+  artistSeriesLoader
 ) {
-  if (!gravityGraphQLLoader || !artistSeriesIDs?.length) return []
+  if (!artistSeriesLoader || !artistSeriesIDs?.length) return []
 
   return Promise.all(
     artistSeriesIDs.map(async (id) => {
-      const data = await gravityGraphQLLoader({
-        query: gql`
-          query GetArtistSeriesLabelsQuery($id: ID!) {
-            artistSeries(id: $id) {
-              title
-            }
-          }
-        `,
-        variables: { id },
-      })
-      const {
-        artistSeries: { title },
-      } = data
+      const { title } = await artistSeriesLoader(id)
```

Comment:
Let it fail! Then it bubbles up and is handled in all our standard ways.

Reply:
(all of this code is defensive that any of these labels can be missing, and does't effect the end result, it's just cleanly left out. Doing `data?.title` may actually lead to null/empty strings being returned somewhere, whereas its better for the exception to be thrown).

Also I don't believe the return of the data loader is actually nullable. B/c its a promise it might reject, but if it resolves it wouldn't be nullable.